aries_cloudagent/multitenant/admin/routes.py

```diff
@@ -306,17 +308,17 @@ async def wallet_create(request: web.BaseRequest):
     if image_url:
         settings["image_url"] = image_url
 
-    async with context.session() as session:
-        try:
+    try:
+        async with context.session() as session:
             multitenant_mgr = session.inject(BaseMultitenantManager)
```

Comment:
BaseMultitenantManager can be injected from the profile. Session should not be created for this